src/components/project/Releases.vue

```diff
@@ -0,0 +1,405 @@
+      v-if="project"
+      class="project-releases__content"
+    >
+      <div
+        v-if="groupedReleases && Object.keys(groupedReleases).length"
+        class="project-releases__list"
+      >
+        <div
+          v-for="(dayReleases, day) in groupedReleases"
+          :key="day"
+          class="project-releases__day"
+        >
+          <div class="project-releases__day-header">{{ formatDayHeader(day) }}</div>
+          <div
+            v-for="(release, index) in dayReleases"
+            :key="release._id"
+            class="project-releases__item"
+            :class="{ 'project-releases__item--expanded': expandedReleases[`${day}-${index}`] }"
+            @click="toggleRelease(`${day}-${index}`)"
+          >
+            <div class="project-releases__item-header">
+              <div class="project-releases__item-info">
+                <div class="project-releases__time">{{ formatTime(release.release) }}</div>
+                <div class="project-releases__name">{{ release._id }}</div>
+              </div>
+              <div class="project-releases__files-count">
+                {{ release.files ? release.files.length : 0 }} files
+              </div>
+            </div>
+            <div v-if="expandedReleases[`${day}-${index}`]" class="project-releases__details">
+              <div class="project-releases__commits" v-if="release.commits && release.commits.length">
+                <div class="project-releases__section-title">Commits</div>
+                <div
+                  v-for="(commit, commitIndex) in release.commits"
+                  :key="commitIndex"
+                  class="project-releases__commit"
+                >
+                  <div class="project-releases__commit-header">
+                    <span class="project-releases__commit-icon">📝</span>
+                    <span class="project-releases__commit-hash">{{ commit.hash.substring(0, 7) }}</span>
+                    <span class="project-releases__commit-author">{{ commit.author }}</span>
+                    <span class="project-releases__commit-date">{{ commit.date | prettyDate }}</span>
+                  </div>
+                  <div class="project-releases__commit-title">{{ commit.title }}</div>
+                </div>
+              </div>
+              <div class="project-releases__files" v-if="release.files && release.files.length">
+                <div class="project-releases__section-title">Source Maps</div>
+                <div
+                  v-for="(file, fileIndex) in release.files"
+                  :key="fileIndex"
+                  class="project-releases__file"
+                >
+                  <span class="project-releases__file-name">{{ file.originFileName }}</span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        v-else
+        class="project-releases__empty"
+      >
+        {{ $t('projects.noReleases') }}
```

Comment:
i18n missed. Use `You do sent any release. Releases helps .... Use following link to know how to setup releases.`